src/java.base/share/classes/java/io/Console.java

```diff
@@ -117,13 +117,13 @@ public PrintWriter writer() {
      * This method is intended to be used by sophisticated applications, for
      * example, a {@link java.util.Scanner} object which utilizes the rich
      * parsing/scanning functionality provided by the {@code Scanner}:
-     * <blockquote><pre>
-     * Console con = System.console();
-     * if (con != null) {
-     *     Scanner sc = new Scanner(con.reader());
-     *     ...
+     * {@snippet lang=java :
+     *         Console con = System.console();
+     *         if (con != null) {
+     *             Scanner sc = new Scanner(con.reader());
+     *             code: // @replace substring="code:" replacement="..."
+     *         }
```

Comment:
4 space indentation would be enough.

Reply:
So changed in b6abb2e40fe574f40ebdfc0c5b1e28ed4b09126e.